org/chapter1.org

```diff
@@ -26,7 +26,7 @@
 
 இன்று, தொழில்நுட்பம் முன்பை விட வலுவாக இருக்கும்போது, ​​இது மிகவும் முக்கியமானது.
 
-சுதந்திரத்தின் கொள்கைகள்(ideology) சி இன் தன்மையில் பிரதிபலிக்கிறது. உங்களிடமிருந்து சிறிதளவு சி மறைக்கிறது, அதன் குறைபாடுகளை. சிறிதளவு சி ​​நீங்கள் செய்வதிலிருந்து தடுக்கிறது, திட்டங்களை சில நேரங்களில் கொடூரமான வழிகளில் உடைக்கிறது. நீங்கள் சி இல் குறியீடு(Code) செய்யும் போது, ​​நீங்கள் ஒரு குறிப்பிட்ட பாதையை மட்டும் பின்பற்றவில்லை, ஆனால் நீங்கள் பல தேர்வுகளை எதிர்கொள்கிறீர்கள், மேலும் சி அந்த முடிவுகளை எடுக்க உங்களுக்கு சவால் விடும்.
```

Comment:
As நிரல் also mean program this can confuse people. So we will stick with குறியீடு if someone come up with better tamil word we can replace it.

Reply:
I have reverted the code back to குறியீ